packages/eslint-plugin/docs/rules/adjacent-overload-signatures.md

```diff
@@ -95,3 +95,9 @@ If you don't care about the general structure of the code, then you will not nee
 - [x] ✅ Recommended
 - [ ] 🔧 Fixable
 - [ ] 💭 Requires type information
+
+### Source
+
+- Rule: [adjacent-overload-signatures.ts](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/rules/adjacent-overload-signatures.ts)
+- Documentation: [adjacent-overload-signatures.md](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/adjacent-overload-signatures.md)
+- Tests: [adjacent-overload-signatures.test.ts](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/tests/rules/adjacent-overload-signatures.test.ts)
```

Comment:
I don't know much about the new pipeline - @JoshuaKGoldberg Is this something we can generate automatically as part of our website build?

Reply:
Hmm probably yeah! I can take a look later this week.

Reply:
@JoshuaKGoldberg Just a heads up on something that I noticed while creating the links: The test files for `indent` and `no-unused-vars` are non-standard (multiple files in a subdirectory) so would need some special-casing.